spec/rails_helper.rb

```diff
@@ -46,6 +53,8 @@ def within(...)
 
 FactoryBot.use_parent_strategy = false
 
+Webdrivers::Chromedriver.required_version = "114.0.5735.90"
```

Comment:
Maybe we could merge the current `master` branch instead, in order to avoid this change? :thinking: